nginx.conf

```diff
@@ -91,6 +91,14 @@ http {
             return 302 {{env "ROOT_REDIRECT_URL"}};
         }
 
+        
+        # Redirect old thumbnail url format
+        #    Feb 2019 - about 1000 requests a month are still done directly to api (and all of them fail with 500 or 404)
+        #    Nov 2019 - still about 1000 requests per month, most return 500 or 404, a few return 301
+        location = /api/image {
+            return 301 https://$host/api/v2/thumbnail-by-url.json$is_args$args;
+        }
+
```

Comment:
I don't think it's necessary to move this, because the equals matches are always checked first before the ~ matches, but there's no harm either.